src/PointwiseFunctions/AnalyticData/Xcts/Binary.hpp

```diff
@@ -0,0 +1,344 @@
+
+ private:
+  void add_deriv_of_window_function(gsl::not_null<tnsr::ijj<DataType, Dim>*>
+                                        deriv_conformal_metric) const noexcept;
+};
+}  // namespace detail
+
+template <typename IsolatedObjectRegistrars, typename Registrars>
+class Binary;
+
+namespace Registrars {
+template <typename IsolatedObjectRegistrars>
+struct Binary {
+  template <typename Registrars>
+  using f = Xcts::AnalyticData::Binary<IsolatedObjectRegistrars, Registrars>;
+};
+}  // namespace Registrars
+
+/*!
+ * \brief Binary compact-object data in general relativity, constructed from
+ * superpositions of two isolated objects.
+ *
+ * This class implements background data for the XCTS equations describing two
+ * objects in a quasi-equilibrium orbit, i.e. with \f$\bar{u}=0\f$ and
+ * \f$\partial_t K=0\f$. Both objects can be chosen from the list of
+ * `IsolatedObjectRegistrars`, e.g. they can be black-hole or neutron-star
+ * solutions in different coordinates. Most quantities are constructed by
+ * superposing the two isolated solutions (see e.g. Eq. (8-9) in
+ * \cite Varma2018sqd or Eq. (45-46) in \cite Lovelace2008tw):
+ *
+ * \f{align}
+ * \bar{\gamma}_{ij} &= f_{ij} + \sum_{\alpha=1}^2
+ * e^{-r_\alpha^2 / w_\alpha^2}\left(\gamma^\alpha_{ij} - f_{ij}\right) \\
+ * K &= \sum_{\alpha=1}^2 e^{-r_\alpha^2 / w_\alpha^2}K^\alpha
+ * \f}
+ *
+ * where \f$\gamma^\alpha_{ij}\f$ and \f$K^\alpha\f$ denote the spatial metric
+ * and extrinsic-curvature trace of the two individual solutions, \f$r_\alpha\f$
+ * is the Euclidean coordinate-distance from the center of each object and
+ * \f$w_\alpha\f$ are parameters describing Gaussian falloff-widths. The
+ * Gaussian falloffs facilitate that the influence of either of the two objects
+ * at the position of the other is strongly damped, and they also avoid
+ * logarithmic scaling of the solution at large distances where we would
+ * typically employ an inverse-radial coordinate map and asymptotically-flat
+ * boundary conditions. The falloff-widths are chosen in terms of the Newtonian
+ * Lagrange points of the two objects in \cite Varma2018sqd and
+ * \cite Lovelace2008tw, and they are input parameters in this implementation.
+ * The falloff can be disabled by passing `std::nullopt` to the constructor, or
+ * `None` in the input file.
+ *
+ * The remaining quantities that this class implements relate to the orbital
+ * motion of the two objects. To obtain initial data in "co-rotating"
+ * coordinates where the two objects are initially at rest we prescribe the
+ * background shift
+ *
+ * \f{equation} \beta^i_\mathrm{background} = (-\Omega y, \Omega x, 0) \f}
+ *
+ * where \f$\Omega\f$ is the angular-velocity parameter.
+ */
+template <typename IsolatedObjectRegistrars,
+          typename Registrars = tmpl::list<
+              Xcts::AnalyticData::Registrars::Binary<IsolatedObjectRegistrars>>>
+class Binary : public ::AnalyticData<3, Registrars> {
```

Comment:
I can't find the documentation for this class. I'm not sure if that's an issue with how I rendered the PR docs. Could you double check that it shows up for you? Maybe you need to `cond` the forward declaration on line 135?

Reply:
It shows up on my machine with Doxygen 1.9.1, but didn't in the rendered docs from the CI workflow. I added the `cond`, maybe it works now :)